index.html

```diff
@@ -149,13 +149,18 @@ <h2 class="primary color1">I'm always interested in hearing about new projects,
             </div>
             <form class="form" action="https://formspree.io/f/mwkjgvpr"
             method="POST">
-                <input type="text" class="mobile-form" required maxlength="30" placeholder="Full name">
+                <input type="text" name="full_name" class="mobile-form" required maxlength="30" placeholder="Full name">
                 <div class="desktop-form">
+                    <input type="text" name="first_name" required maxlength="30" placeholder="First Name">
+                    <input type="text" name="last_name" required maxlength="30" placeholder="Last Name">
                     <input type="text" name="full_name" required maxlength="30" placeholder="Full Name">
                 </div>
                 <input type="email" name="email" required maxlength="30" placeholder="Email Address">
                 <textarea type="text" name="message" required class="textarea" maxlength="500" placeholder="Hey I'd like to connect too"></textarea>
-                <button type="submit" class="get-in-touch-btn" name="get-in-touch" id="">Get in touch</button>
+                <div class="validation-message-container">
+                    <p class="validation-message"></p>
+                </div>
+                <button type="submit" class="get-in-touch-btn" name="submit">Get in touch</button>
```

Comment:
- [ ] When I reload the page  information should be persistent

Reply:
- [ ] After filling out the form, and not submitting it, the entered values should be saved in the browser but currently its not saved

Reply:
@DeliceLydia I am sorry but I don't understand, it is working properly in my machine. Could you please explain a bit more.